plugin-base/src/main/java/io/everytrade/server/plugin/impl/everytrade/BlockchainBtcConnector.java

```diff
@@ -44,26 +68,49 @@
         "Blockchain BTC Connector",
         "",
         SupportedExchange.BLOCKCHAIN.getInternalId(),
-        List.of(PARAMETER_ADDRESS, PARAMETER_FIAT_CURRENCY, PARAMETER_IS_WITH_FEE)
+        List.of(
+            PARAMETER_ADDRESS,
+            PARAMETER_FIAT_CURRENCY,
+            PARAMETER_IS_IMPORT_BUY,
+            PARAMETER_IS_IMPORT_SELL,
+            PARAMETER_IS_BUY_WITH_FEE,
+            PARAMETER_IS_SELL_WITH_FEE
+        )
     );
 
     private static final String CRYPTO_CURRENCY = "BTC";
     private final String source;
     private final String fiatCurrency;
-    private final String isWithFee;
+    private final String isImportBuy;
```

Comment:
Never ever name variables with "is" on the beginning. is should be added only to methods that return boolean.
rename:
isImportBuy to importDepositsAsBuys
isImportSell to importWithdrawalsAsBuys
isBuyWithFee to importFeesFromDeposits
isSellWithFee to importFeesFromWithdrawals
etc.